skimage/_shared/interpolation.pxd

```diff
@@ -28,62 +32,65 @@ cdef inline Py_ssize_t fmax(Py_ssize_t one, Py_ssize_t two) nogil:
 cdef inline Py_ssize_t fmin(Py_ssize_t one, Py_ssize_t two) nogil:
     return one if one < two else two
 
+ctypedef fused np_real_numeric_out:
```

Comment:
What's happening here?

Reply:
I'm forcing cross type compilation too. so uint8->uint16, but more importantly uint8->float64

Reply:
I added a note about this.